src/main/java/uk/gov/ons/ctp/integration/mock/endpoints/HelpEndpoint.java

```diff
@@ -1,21 +1,17 @@
 package uk.gov.ons.ctp.integration.mock.endpoints;
```

Comment:
Something to consider would be splitting up the help handling into 2 files, with one for the case stuff and the other for the ai specific stuff. Might make it all a bit less intertwined. 
I agree with Robs comment about a top level /help endpoint (which I assume would basically direct you to the /addresses/help and cases equivalent).

Reply:
After fixing and tidying, I think it may be OK left as a single file. See what you think.